src/DotNetWorker.Grpc/GrpcStartupOptions.cs

```diff
@@ -17,5 +17,7 @@ internal class GrpcWorkerStartupOptions
         public int GrpcMaxMessageLength { get; set; }
 
         public string? ScriptRoot { get; set; }
+
+        public string? AzureWebJobsFeatureFlags { get; set; }
```

Comment:
Will HOST send this value? Where does host get this value from? Is it a comma separated string of feature names? Can you share how a sample value look like?

Reply:
This is set in `local.settings.json` on the worker side, or if you're running from the host solution, you can add it to `launchsettings.json`. For our purposes, you just need `"AzureWebJobsFeatureFlags":"EnableWorkerIndexing"` but you can also have multiple features separated by a semicolon.

Reply:
Is the reason for the worker to care about this an optimization so we kick off the task early as opposed to deferring to when the request is sent?